docs/api.rst

```diff
@@ -1227,6 +1227,60 @@ It's also possible to embed `Materialized Views <https://www.postgresql.org/docs
 
   If view definitions change you must refresh PostgREST's schema cache for this to work properly. See the section :ref:`schema_reloading`.
 
+Embedding Views Containing Joins
+--------------------------------
+
+The same embedding rules applies to views with joined tables, that is, the column of any table participating in the join with a **foreign key** definition must be present in the view.
+
+For instance, let's create a view called ``film_competition_view`` that will join three tables:
+
+.. code-block:: postgres
+
+  CREATE VIEW film_competition_view AS
+  SELECT
+   films.title as film_title
+   , competitions.name as competition_name
+   , film_id
+  FROM
+   nominations
+   JOIN films ON films.id = nominations.film_id
+   JOIN competitions ON competitions.id = nominations.competition_id;
+
+The view can be embedded with the table ``films`` because it has the ``film_id`` column. But what if you want to embed it with the ``actors`` table, for instance? The ``film_id`` column alone will not allow it since it belongs to ``nominations`` and there is no relationship that can be inferred between that table and ``actors``. In this case, you need to select the ``film.id`` column which will allow any relationship that worked with the ``films`` table (using ``film.id`` as a foreign key) to also work for the view.
```

Comment:
```suggestion
The view can be embedded with the table ``films`` because it has the ``film_id`` column.

But what if you want to embed it with the ``actors`` table? The ``film_id`` column alone will not allow it since it belongs to ``nominations``, which doesn't have a relationship with ``actors``. In this case, you need to select the ``film.id`` column, that comes from ``films`` which has a many-to-many relationship with ``actors``.
```

Reply:
```suggestion
The view can be embedded with the table ``films`` because it has the ``film_id`` column, which comes from ``nominations`` and has a relationship with ``films``.

But what if you want to embed it with the ``actors`` table? The ``film_id`` column alone will not allow it since it belongs to ``nominations``, which doesn't have a relationship to``actors``. In this case, you need to select the ``film.id`` column, that comes from ``films`` and has a many-to-many relationship with ``actors``.
```